src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+.body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 42px;
+}
+
+.logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+  margin-right: 42px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
```

```suggestion
.nav__item:not(:first-childe) {
```
И тогда строки 30-32 можно удалить